cmd/gaia/init/testnet.go

```diff
@@ -173,8 +190,27 @@ func testnetWithConfig(config *cfg.Config, cdc *codec.Codec, appInit server.AppI
 		}
 	}
 
+	// Generate empty genesis.json
+	appGenState := app.DefaultGenesisState()
+	appGenState.Accounts = accs
+	appGenStateJSON, err := codec.MarshalJSONIndent(cdc, appGenState)
+	if err != nil {
+		return err
+	}
+	genDoc := types.GenesisDoc{
```

Comment:
suggest using `NewGenesisDoc`

Reply:
Constructor is not available in tendermint, and that comes from there

Reply:
yeah fair then makes sense to leave